### DISCUSSION TITLE
mPDF not found - Server error 500

### DISCUSSION THREAD

**Discussion by wade-lich** [opendcim/openDCIM#1370] [Q&A] [answered]
Fatal error: Uncaught Error: Class "mPDF" not found in /var/www/devdcim/report_supply_status.php:17

This is also an issue on the demo site.  Is there an easy code fix for this?  

Thanks for the help as always.

**Comment by samilliken** [comment, marked as answer]
Opened an issue for this one [#1375](https://github.com/opendcim/openDCIM/issues/1375)
